src/Api/Jobs/JobsHostedService.cs

```diff
@@ -23,6 +23,14 @@ public override async Task StartAsync(CancellationToken cancellationToken)
                 .StartNow()
                 .WithCronSchedule("0 0 * * * ?")
                 .Build();
+            var emergencyAccessNotificationTrigger = TriggerBuilder.Create()
+                .StartNow()
+                .WithCronSchedule("0 * * * * ?")
+                .Build();
+            var emergencyAccessTimeoutTrigger  = TriggerBuilder.Create()
+                .StartNow()
+                .WithCronSchedule("0 * * * * ?")
+                .Build();
```

Comment:
I prefer these Crons to be named after the time threshold they represent (like the others). For example, `everySecondTrigger`, `everyOneMinuteTrigger`.

Reply:
I thought about using the same naming pattern, but the triggers are identical and unfortunately I couldn't find a way to re-use the same trigger multiple times. We could name them `everyHourTrigger1` or something.

Reply:
You can't reuse a trigger?

Reply:
No, I get the following error when I use the same trigger on multiple jobs:
```
Unhandled exception. Quartz.SchedulerException: Trigger does not reference given job!
   at Quartz.Core.QuartzScheduler.ScheduleJob(IJobDetail jobDetail, ITrigger trigger, CancellationToken cancellationToken)
   at Bit.Core.Jobs.BaseJobsHostedService.StartAsync(CancellationToken cancellationToken) in C:\Users\Oscar\Documents\Projects\Bitwarden\server\src\Core\Jobs\BaseJobsHostedService.cs:line 50
   at Bit.Api.Jobs.JobsHostedService.StartAsync(CancellationToken cancellationToken) in C:\Users\Oscar\Documents\Projects\Bitwarden\server\src\Api\Jobs\JobsHostedService.cs:line 52
```

And according to https://stackoverflow.com/a/2776842 this seems to be the expected behaviour, workarounds are to ether have a job that triggers other jobs or identical triggers.